xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -700,6 +702,50 @@ namespace PVR
                                             CVariant{timer->Title()});
   }
 
+  class AsyncRecordingAction : private IRunnable
+  {
+  public:
+    bool Execute(const CFileItemPtr &item = CFileItemPtr());
```

Comment:
> A default value seems unnecessary?

https://github.com/xbmc/xbmc/pull/11863/files#diff-e47a4eb266b6b884e4b669666a1cc89cR814

Alternatives would be Execute(CFileItemPtr()) or overloaded Execute().